src/atomate2/common/schemas/pheasy.py

```diff
@@ -0,0 +1,812 @@
+        "formula unit for temperatures in temperature_list",
+    )
+
+    heat_capacities: Optional[list[float]] = Field(
+        None,
+        description="heat capacities in J/K/mol per "
+        "formula unit for temperatures in temperature_list",
+    )
+
+    internal_energies: Optional[list[float]] = Field(
+        None,
+        description="internal energies in J/mol per "
+        "formula unit for temperatures in temperature_list",
+    )
+    entropies: Optional[list[float]] = Field(
+        None,
+        description="entropies in J/(K*mol) per formula unit"
+        "for temperatures in temperature_list ",
+    )
+
+    temperatures: Optional[list[int]] = Field(
+        None,
+        description="temperatures at which the vibrational"
+        " part of the free energies"
+        " and other properties have been computed",
+    )
+
+    total_dft_energy: Optional[float] = Field("total DFT energy per formula unit in eV")
+
+    has_imaginary_modes: Optional[bool] = Field(
+        None, description="if true, structure has imaginary modes"
+    )
+
+    # needed, e.g. to compute Grueneisen parameter etc
+    force_constants: Optional[Forceconstants] = Field(
+        None, description="Force constants between every pair of atoms in the structure"
+    )
+
+    born: Optional[list[Matrix3D]] = Field(
+        None,
+        description="born charges as computed from phonopy. Only for symmetrically "
+        "different atoms",
+    )
+
+    epsilon_static: Optional[Matrix3D] = Field(
+        None, description="The high-frequency dielectric constant"
+    )
+
+    supercell_matrix: Matrix3D = Field("matrix describing the supercell")
+    primitive_matrix: Matrix3D = Field(
+        "matrix describing relationship to primitive cell"
+    )
+
+    code: str = Field("String describing the code for the computation")
+
+    phonopy_settings: PhononComputationalSettings = Field(
+        "Field including settings for Phonopy"
+    )
+
+    thermal_displacement_data: Optional[ThermalDisplacementData] = Field(
+        "Includes all data of the computation of the thermal displacements"
+    )
+
+    jobdirs: Optional[PhononJobDirs] = Field(
+        "Field including all relevant job directories"
+    )
+
+    uuids: Optional[PhononUUIDs] = Field("Field including all relevant uuids")
+
+    @classmethod
+    def from_forces_born(
+        cls,
+        structure: Structure,
+        supercell_matrix: np.array,
+        displacement: float,
+        sym_reduce: bool,
+        symprec: float,
+        use_symmetrized_structure: Union[str, None],
+        kpath_scheme: str,
+        code: str,
+        mp_id: str,
+        displacement_data: dict[str, list],
+        total_dft_energy: float,
+        epsilon_static: Matrix3D = None,
+        born: Matrix3D = None,
+        **kwargs,
+    ) -> Self:
+        """Generate collection of phonon data.
+
+        Parameters
```

Comment:
while this is all working, could you simply try to reuse more from the schemas/phonopy.py code? This is mostly duplicated at the moment.

Reply:
Did it.